tests/userSocialMedia.test.ts

```diff
@@ -0,0 +1,121 @@
+import faker = require('faker');
+import { UserModel } from '../models/UserModel';
+import { SocialMediaType } from '../types';
+import { ControllerFactory } from './controllers';
+import { DatabaseConnection, PortalState, UserFactory } from './data';
+import { UserSocialMediaFactory } from './data/UserSocialMediaFactory';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('social media URL submission', () => {
+  test('properly persists on successful submission', async () => {
+    const conn = await DatabaseConnection.get();
+    let member = UserFactory.fake();
+    const userSocialMedia = UserSocialMediaFactory.fake({ user: member, type: SocialMediaType.FACEBOOK });
```

Comment:
nit: the type of social media doesn't matter here so we can omit specifying the type here and just let the factory randomly generate a type for us. I think the type only matters for the test below this one where we verify that two social medias of the same type can't be created, so for the other tests, let's also randomize the types there.